core/src/main/java/com/linecorp/armeria/common/DefaultContextAwareEventLoop.java

```diff
@@ -170,7 +172,7 @@ public ChannelFuture register(Channel channel,
     public String toString() {
         return MoreObjects.toStringHelper(this)
                           .add("eventLoop", eventLoop)
-                          .add("context", context())
+                          .add("context", contextOrNull())
```

Comment:
Not necessarily your fault, but how about making the order of properties consistent with `DefauiltContextAwareExecutorService`? i.e.

```
                          .add("context", contextOrNull())
                          .add("eventLoop", eventLoop)
```